starter-code/src/index.js

```diff
@@ -1,8 +1,47 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import App from "./App.js";
+import "./App.css";
 
 ReactDOM.render(
-  <App />,
+  <section className = "partone">
```

Comment:
all of this code (apart from ``` <App /> ```) should be in App.js. With a few exceptions, we don't add any code to our index.js once we've set it up. You'll learn more about this over the next few lessons